src/cargo/core/manifest.rs

```diff
@@ -504,6 +507,10 @@ impl Manifest {
                 })?;
         }
 
+        if self.rust_version.is_some() {
+            self.unstable_features.require(Feature::rust_version())?;
+        }
+
```

Comment:
```suggestion
```

I don't think this is necessary, since it is already checked in `to_real_manifest`.